.github/workflows/run-bats-tests.yml

```diff
@@ -49,3 +50,38 @@ jobs:
       - name: Execute Test Cases
         run: |
           REPO_DIR="./" CLEAN_UP_TEST_ENVIRONMENT=true ./bats/lib/bats/bin/bats ./bats/tests
+
+      # Send the result to Slack when a PR is merged and tests were executed regularly
+      - name: Notify success
+        if: ${{ success() && contains(fromJson('["push", "schedule"]'), github.event_name) }}
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: ${{ secrets.CHANNEL_ID_FOR_MESSAGES }}
```

Comment:
Actually channel IDs are not a secret as it can be shown as a part of Slack URL. Should I hard code instead using secrets? 
Using secrets lets us change the channel to notify a bit more easily, but it would be harder to understand what's going on. Either is fine.

Reply:
I think it's better not to use secrets if it's not a secret.

Reply:
@erwin1 Thank you Erwin. Karol and I talked about it, and it is better not to hardcode.
I thought it is not a secret, but only people in Slack can see these IDs (not completely public). Using secrets seems safer.